cfgov/cfgov/settings/base.py

```diff
@@ -1,6 +1,7 @@
 import os
 from unipath import Path
 
+DEBUG=True
```

Comment:
With https://github.com/cfpb/django-sheerlike/pull/12, this should not be turned on.